source/scripting/Ped.cs

```diff
@@ -1555,6 +1555,10 @@ public void OpenParachute()
 		{
 			Function.Call(Hash.FORCE_PED_TO_OPEN_PARACHUTE, Handle);
 		}
+		public void ParachuteTo(Vector3 location)
+		{
+		    	Function.Call(Hash.SET_PARACHUTE_TASK_TARGET, Handle, location.X, location.Y, location.Z);
```

Comment:
Should i name it differently... ?
This is to be used outside of a Task sequence, to update the position of the task.
Such as....
new Sequence()->Tasks.ParachuteTo(moving.entity.position)->PerformSequence();

OnTick->If Sequence Active->Ped.ParachuteTo(moving.entity.position);

Then the task sequence is uninterrupted but the ped's parachute target is updated per the moving targets position.